src/unix/fs.c

```diff
@@ -819,7 +819,7 @@ static ssize_t uv__fs_copyfile(uv_fs_t* req) {
                      &fs_req,
                      req->new_path,
                      dst_flags,
-                     statsbuf.st_mode,
+                     0,
```

Comment:
Isn't this insecure for a new file when the eventual permissions are narrower than the default ones?  Another process would be able to open it in the time gap between creation and fchmod().

Reply:
I think you're right. I've rolled back that part of the change.